### DISCUSSION TITLE
Is anyone else besides me waiting for the 5.3 version?

### DISCUSSION THREAD

**Discussion by gustavorios2929** [Sixze/ALS-Refactored#352] [Polls]
Is anyone else besides me waiting for the 5.3 version?

**Comment by Birusama** [comment]
There is already a 5.3 branch, it's just not merged into main yet: https://github.com/Sixze/ALS-Refactored/tree/ue-5.3

**Reply by gustavorios2929** [reply]
I didn't know that! Thank you very much!

**Reply by dungeon2567** [reply]
When will it get merged? is there any cons?

**Comment by SmoothCriminal163** [comment]
> There is already a 5.3 branch, it's just not merged into main yet: https://github.com/Sixze/ALS-Refactored/tree/ue-5.3
The link is dead

**Reply by 4re** [reply]
Because it has been merged into main.
